templates/zerver/app/left_sidebar.html

```diff
@@ -60,7 +60,9 @@
         </ul>
         <div id="streams_list" class="zoom-out">
             <div id="streams_header" class="zoom-in-hide"><h4 class="sidebar-title tippy-zulip-tooltip" data-tippy-content="{{ _('Filter streams') }} (q)">{{ _('STREAMS') }}</h4>
-                <i id="streams_inline_cog" class='fa fa-cog tippy-zulip-tooltip' aria-hidden="true" data-tippy-content="{{ _('Subscribe, add, or configure streams') }}"></i>
+                <span class="tippy-zulip-tooltip streams_inline_cog_wrapper" data-tippy-content="{{ _('Add streams') }}" data-tippy-placement="top">
```

Comment:
Same element cannot have two tippy instaces, so I had to add a wrapper element for the tooltip.